torch/fx/traceback.py

```diff
@@ -192,7 +192,10 @@ def _from_dict(cls, d: Optional[dict]) -> Optional["NodeSource"]:
             node_source.node_info = None
 
         # Recursively deserialize nested from_node
-        node_source.from_node = [cls._from_dict(fn) for fn in d.get("from_node", [])]
+        if d.get("from_node", None) is not None:
```

Comment:
You should assign the fetch to a var before refining, it will more reliably work that way